setup.py

```diff
@@ -0,0 +1,39 @@
+import os
+from glob import glob
+from os.path import basename
+from os.path import splitext
+
+from setuptools import find_packages, setup
+
+
+def read(file_name):
+    return open(os.path.join(os.path.dirname(__file__), file_name)).read()
+
+
+
+setup(
+    name='sagemaker-container-support',
```

Comment:
Name should match our existing name: https://github.com/aws/sagemaker-container-support/blob/master/setup.py#L16

so sagemaker_container_support

I know it doesn't follow python convention, but I think if we're keeping the same name, then continuity is important. (If we decide to change the name, everything changes, of course.)